code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -299,3 +302,71 @@
 	GUN_UPGRADE_DAMAGE_PSY = 0.4)
 	I.req_gun_tags = list(GUN_PROJECTILE)
 	I.gun_loc_tag = GUN_MECHANISM
+
+/obj/item/weapon/gun_upgrade/barrel/gauss
+	name = "Syndicate \"Gauss Coil\" barrel"
+	desc = "Make bullet pierce through wall and penetrate armors easily, but losing rate of fire and increece recoil."
+	icon_state = "Gauss"
+
+/obj/item/weapon/gun_upgrade/barrel/gauss/New()
+	..()
+	var/datum/component/item_upgrade/I = AddComponent(/datum/component/item_upgrade)
+	I.weapon_upgrades = list(
+		GUN_UPGRADE_DAMAGE_BRUTE = 10,
+		GUN_UPGRADE_PEN_MULT = 1.5,
+		GUN_UPGRADE_PIERC_MULT = 2,
+		GUN_UPGRADE_FIRE_DELAY_MULT = 1.7,
+		GUN_UPGRADE_RECOIL = 1.2,
```

Comment:
All of those numbers should be lowered to affect gun less. That's too much.

Reply:
Changed